docs/resources/workspace_desktop.md

```diff
@@ -200,7 +215,7 @@ resource "huaweicloud_workspace_desktop" "test" {
 
   lifecycle {
     ignore_changes = [
-      user_email, nic,
+      user_email, "power_action","power_action_type",
```

Comment:
```suggestion
      user_email, power_action, power_action_type,
```
The quotation marks here are superfluous

Reply:
fixed